### PR TITLE
Using npm module

### PR DESCRIPTION
**Pull request by DKbyo**
- Deleted files that npm espina provides
- Modify the location and session reducer in order to use the new router

### CODE REVIEW

File: shared/lib/routes.js [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
+    path:`/:locale?/${i18n.t('details')}/:example_id`,
+    url: function(action,i18n,payload,params) {
+      return `/${i18n.language}/${i18n.t('details')}/${payload.id}`
+    }
+  },{
+    name:"Login",
+    path:`/:locale?/${i18n.t('login')}`
+  },{
+    name:"Missing",
+    path:"/"
```
Comment (manue1) [inline shared/lib/routes.js]: The 'Missing' component should appear when there's no other route match, so it should have a wildcard as its path, right? Currently when navigating to a non-existent route an error is thrown `TypeError: Cannot read property 'name' of undefined` by the router.

Comment (DKbyo) [reply shared/lib/routes.js]: Now route can accept a regex in path

**Comment by erichulburd** [comment]
When I run this I get:

```
entry.js:5Uncaught TypeError: (0 , _history.useQueries) is not a function(…)
(anonymous function)    @   entry.js:5
__webpack_require__ @   bootstrap 8db6925…:19
(anonymous function)    @   bootstrap 8db6925…:39
(anonymous function)    @   bootstrap 8db6925…:39
```

Is this due to my updates in package.json config? Or is this a problem with the router npm module?

**Comment by erichulburd** [comment]
Let's also get tests passing before this is merged.

**Comment by DKbyo** [comment]
Yes... we need to use 

``` js
import createHistory from 'history/createBrowserHistory'
```

Because useQueries is deprecated

**Comment by erichulburd** [comment]
Cool now working.

**Pull request merged**